Samples/word-add-in-get-set-edit-openxml/README.md

```diff
@@ -0,0 +1,113 @@
+
+- Word on Windows, Mac, and in a browser.
+
+## Prerequisites
+
+- Visual Studio 2019 or later with the **Office/SharePoint development** workload installed.
+- Microsoft 365 - You can get a [free developer sandbox](https://developer.microsoft.com/microsoft-365/dev-program#Subscription) that provides a renewable 90-day Microsoft 365 E5 developer subscription.
+
+## Run the sample
+
+1. Open the **WD_OpenXML_js.sln** file in Visual Studio 2019 or later.
+2. Choose **Debug > Start Debugging**, or choose the F5 key.
+3. On the **Home** tab, in the **OOXML Editor** group, choose **Open**.
+
+The SampleDoc.docx file will open in Word since it is set as the **StartAction** property of the add-in. The document contains a range of common rich content types for Word, including the following:
+
+- formatted text
+- styled text
+- a formatted image
+- a text box using WordArt formatting
+- an Office drawing shape
+- a content control that can be used for binding to a specific location in the document
+- a formatted table
+- a styled table
+- a dynamic SmartArt graphic
+- a chart
+
+The following screenshot (Figures 1a and Figure 1b) show how the document surface appears when the solution starts and the SampleDoc.docx file is opened.
+
+Figure 1a. SampleDoc.docx document contents.
+
+![Screen shot of SampleDoc.docx document opened with images and formatted text.](/description/CG_GetSetOOXML_fig01a.gif)
+
+Figure 1b. More SampleDoc.docx document contents.
+
+![Screen shot of SampleDoc.docx document opened with tables and SmartArt.](/description/CG_GetSetOOXML_fig01b.gif)
+
+**Note**
+
+The sample uses Office Open XML (the ooxml coercion type) rather than HTML or plain text because OOXML coercion enables you to interact with virtually every type of content a user can insert in their document, such as in the examples you see in Figures 1a - 1b.
+
+Figure 2. Task pane appearance after using the Get… button to extract Office Open XML for selected content
+
+![Screenshot of the add-ins task pane opened in Word](/description/add-in-screenshot.png)
+
+When you select content and choose the 'Get…' button, the add-in uses the JavaScript **getSelectedDataAsync** method to generate a complete Office Open XML document package that includes the selected document content, and places it in the text area of the task pane (as shown in Figure 2).You can use the text area in that task pane as a scratch pad. Copy the Office Open XML markup you retrieve into an XML file, edit it to include just the information you need, and paste it back into the text area of the task pane to test your edited markup.
+
+After you paste or edit the content in the text area of the task pane, you can click in a blank area of the document and then choose the **Insert…** button on the task pane to test the integrity of your markup. That button uses the JavaScript **setSelectedDataAsync** method to insert the contents of the task pane text area as rich content in Word, using the ooxml coercion type.
```

Comment:
```suggestion
After you paste or edit the content in the text area of the task pane, click in a blank area of the document and then choose **Insert…** on the task pane to test the integrity of your markup. That button uses the JavaScript **setSelectedDataAsync** method to insert the contents of the task pane text area as rich content in Word, using the OOXML coercion type.
```

Reply:
In addition to Linda Lu's suggestions, suggest using ``. Recommend keeping the "the Insert button" text since you refer to that button later.
```suggestion
After you paste or edit the content in the text area of the task pane, click in a blank area of the document and then choose the **Insert…** button on the task pane to test the integrity of your markup. That button uses the JavaScript `setSelectedDataAsync` method to insert the contents of the task pane text area as rich content in Word, using the OOXML coercion type.
```